src/kwok_cluster/mod.rs

```diff
@@ -0,0 +1,132 @@
+use testcontainers::{core::WaitFor, Image};
+
+const NAME: &str = "registry.k8s.io/kwok/cluster";
+const TAG: &str = "v0.4.0-k8s.v1.28.0";
```

Comment:
Maybe it would be useful to upgrade this to v0.5.2 before releasing this module. I know it has been released quite recently, I just thought it might be a nice idea to start up to date :)

Reply:
I pushed it :)